### ISSUE TITLE
Request for a noob friendly video guide on the sm64 Port

### ISSUE THREAD

**Issue by Danibrit1159** [mkst/sm64-port#59]
I'm not very good at compiling code or c++, but I'm very good at watching a video and following it steps visually so I was wondering if y'all can make a video guide it will be very helpful thank you

**Comment by LukaTV939** [comment]
Just do what the guide says, copy and paste the given commands and you should be good. However you need to know where is it built which even I myslef don't know

**Comment by GartFart** [comment]
no, I really do need a video guide. This thing is so confusing and I can't get it to work at all
